feature-libs/asm/components/customer-selection/customer-selection.component.html

```diff
@@ -63,9 +63,12 @@
     </ng-container>
     <ng-container *cxFeatureLevel="'6.1'">
       {{ 'asm.customerSearch.noMatchResult' | cxTranslate }}
-      <a #createCustomerLink class="link" (click)="createCustomer()">{{
```

Comment:
Adding an anchor with a different click handler inside an existing button will cause issues with accessibility and can be confusing to the user. so remove this link.